index.html

```diff
@@ -4,18 +4,142 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <!-- Add a link to your custom CSS here -->
-
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Questrial&family=Raleway&family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
-</head>
 
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>                
+     
+</head>
 <body>
+    <header> 
+        <div class="order">
+            <a href="#" class="nav-link">
+               
+            <a href='#'><p>Order Online</p></a>
+        </div>
+        <div class="bar1">
+           <a class="login" href="#">Log In</a> 
+                <i class='fa fa-user-circle-o' aria-hidden="true">
+                </i> 
+            <a href='#'><p class="h1" >Juice Bar</p></a>
+        
+        <section class="section">
+            <nav class="navbar">
+                <!--<a href="#" class="nav-logo">WebDev.</a>-->
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <a href="#" class="nav-link">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link">Story</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link">Menu</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link">Order Online</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link">Locations & Hours</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link">
+                            <i class='fa fa-user-circle-o' aria-hidden="true">
+                            </i> Log In</a>
+                    </li>
+                </ul>
+                <div class="hamburger">
```

Comment:
When you inspect this div in the Accessibility API, what does it show? How could someone interact with this button if they couldn't see it or access it with a mouse? Take a look and have a think! 😉 

https://developer.chrome.com/blog/full-accessibility-tree/#what-is-the-accessibility-tree